goaviatrix/client.go

```diff
@@ -254,28 +342,28 @@ func (c *Client) PostFile(path string, params map[string]string, files []File) (
 
 			file, err := os.Open(f.Path)
 			if err != nil {
-				return nil, err
+				return nil, "", err
```

Comment:
Is "PostFile" used anywhere else? Output format changing causes issue wherever else it is used, right?

Reply:
I did not change the output format of PostFile. This is actually a new function `func encodeMultipartFormData(params map[string]string, files []File) (*bytes.Buffer, string, error) {` that encapsulates some behavior from PostFile so that I can reuse it in PostFile and PostFileContext